### DISCUSSION TITLE
Whoa there! You have exceeded a secondary rate limit.  Please wait a few minutes before you try again; in some cases this may take up to an hour.  Contact Support — GitHub Status — @githubstatus

### DISCUSSION THREAD

**Discussion by go1-D** [community/community#142378] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Whoa there!
You have exceeded a secondary rate limit.

Please wait a few minutes before you try again;
in some cases this may take up to an hour.

[Contact Support](https://support.github.com/contact) — [GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)

**Comment by lin-ycv** [comment]
if you did a search you'll see that there already a bunch of topics on this, without providing more information on how/what you were doing to recieve this no one can offer you pointers.
https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+secondary+limit

**Reply by garry-ut99** [reply]
Yes, a duplicate of:
- #129998
- #141004
- #141073
- #141641
- #141770

**Comment by Dragodraki** [comment]
Please continue here (Github Support told me it should be fixed)
https://github.com/orgs/community/discussions/141073#discussioncomment-11168607
If the problem persists, please answer in this issue!
